java-checks/pom.xml

```diff
@@ -192,6 +192,16 @@
                   <artifactId>spring-boot-autoconfigure</artifactId>
                   <version>2.0.2.RELEASE</version>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.springframework.security</groupId>
+                  <artifactId>spring-security-crypto</artifactId>
+                  <version>5.0.6.RELEASE</version>
+                </artifactItem>
+                <artifactItem>
+                  <groupId>org.springframework.security</groupId>
+                  <artifactId>spring-security-core</artifactId>
+                  <version>4.2.6.RELEASE</version>
```

Comment:
Why not using the latest version is 5.0.6.RELEASE ?

Reply:
ShaPasswordEncoder and Md5PasswordEncoder have been removed from Spring Security 5.x